### PR TITLE
Merge dev-pt in main 

### PR DESCRIPTION
**Pull request by edsoncelio**
Changes
* Fix links to Portuguese content
* Add three new definitions translated in Portuguese: abstraction, agile_software_development and auto_scalling.

### CODE REVIEW

**Review by jihoon-seo — APPROVED** [review]
Preview: https://deploy-preview-789--cncfglossary.netlify.app/pt-br/

I checked some pt-br pages, and they LGTM. 😊

File: content/pt-br/continuous_delivery.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ A entrega contínua, muitas vezes conhecida como CD, é um conjunto de práticas
...
 
 ## Como isso ajuda
-As estratégias de entrega contínua criam um caminho totalmente automatizado para a produção que testa e implanta o software usando várias estratégias de implantação, como versões [canary](/canary_deployment/) ou [blue-green](/blue_green_deployment/). Isso permite que os desenvolvedores implantem o código com frequência, dando a tranquilidade de que a nova revisão foi testada. Normalmente, o desenvolvimento *trunk-based* é usado em estratégias de entrega contínua, em oposição aos recursos de *branch* ou *pull requests*.
+As estratégias de entrega contínua criam um caminho totalmente automatizado para a produção que testa e implanta o software usando várias estratégias de implantação, como versões [canary](/pt-br/canary_deployment/) ou [blue-green](/blue_green_deployment/). Isso permite que os desenvolvedores implantem o código com frequência, dando a tranquilidade de que a nova revisão foi testada. Normalmente, o desenvolvimento *trunk-based* é usado em estratégias de entrega contínua, em oposição aos recursos de *branch* ou *pull requests*.
```
Comment (jessicalins) [inline content/pt-br/continuous_delivery.md:14]:
```suggestion
As estratégias de entrega contínua criam um caminho totalmente automatizado para a produção que testa e implanta o software usando várias estratégias de implantação, como versões [canary](/pt-br/canary_deployment/) ou [blue-green](/pt-br/blue_green_deployment/). Isso permite que os desenvolvedores implantem o código com frequência, dando a tranquilidade de que a nova revisão foi testada. Normalmente, o desenvolvimento *trunk-based* é usado em estratégias de entrega contínua, em oposição aos recursos de *branch* ou *pull requests*.
```

File: content/pt-br/bare_metal_machine.md [hunk 2]
```diff
@@ -16,4 +16,4 @@ Instalar um sistema operacional em um computador físico é o padrão original d
...
+No contexto de [aplicativos nativos da nuvem](/pt-br/cloud_native_apps/), geralmente pensamos em desempenho em termos de [escala](/scalability/) para um grande número de eventos simultâneos, que podem ser tratados por [escala horizontal](/horizontal_scaling/) (adicionando mais máquinas ao  recursos disponíveis). Mas algumas cargas de trabalho podem exigir um redimensionamento vertical (adicionando mais energia a uma máquina física existente) e/ou uma resposta extremamente rápida de um hardware físico, caso em que o bare metal é mais adequado. Bare metal também permite que você ajuste o hardware físico e possivelmente até os drivers do hardware para melhorar a realização da sua tarefa.
```
Comment (jessicalins) [inline content/pt-br/bare_metal_machine.md:19]:
```suggestion
No contexto de [aplicações nativas em nuvem](/pt-br/cloud_native_apps/), geralmente pensamos em desempenho em termos de [escala](/scalability/) para um grande número de eventos simultâneos, que podem ser tratados por [escala horizontal](/horizontal_scaling/) (adicionando mais máquinas ao  recursos disponíveis). Mas algumas cargas de trabalho podem exigir um redimensionamento vertical (adicionando mais energia a uma máquina física existente) e/ou uma resposta extremamente rápida de um hardware físico, caso em que o bare metal é mais adequado. Bare metal também permite que você ajuste o hardware físico e possivelmente até os drivers do hardware para melhorar a realização da sua tarefa.
```

File: content/pt-br/cloud_native_security.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ category: conceito
 
 ## O que é
 
-A segurança nativa da nuvem é uma abordagem que transforma a segurança em [aplicativos nativos da nuvem](/cloud_native_apps/). Isso garante que a segurança faça parte de todo o ciclo de vida do aplicativo, desde o desenvolvimento até a produção. A segurança nativa da nuvem busca garantir os mesmos padrões que os modelos de segurança tradicionais, enquanto se adapta aos detalhes dos ambientes nativos da nuvem, ou seja, mudanças rápidas de código e infraestrutura altamente efêmera. A segurança nativa da nuvem está altamente relacionada à prática chamada [DevSecOps](/devsecops/).
+A segurança nativa da nuvem é uma abordagem que transforma a segurança em [aplicativos nativos da nuvem](/pt-br/cloud_native_apps/). Isso garante que a segurança faça parte de todo o ciclo de vida do aplicativo, desde o desenvolvimento até a produção. A segurança nativa da nuvem busca garantir os mesmos padrões que os modelos de segurança tradicionais, enquanto se adapta aos detalhes dos ambientes nativos da nuvem, ou seja, mudanças rápidas de código e infraestrutura altamente efêmera. A segurança nativa da nuvem está altamente relacionada à prática chamada [DevSecOps](/devsecops/).
```
Comment (jessicalins) [inline content/pt-br/cloud_native_security.md:9]:
```suggestion
A segurança nativa da nuvem é uma abordagem que transforma a segurança em [aplicações nativas em nuvem](/pt-br/cloud_native_apps/). Isso garante que a segurança faça parte de todo o ciclo de vida do aplicativo, desde o desenvolvimento até a produção. A segurança nativa da nuvem busca garantir os mesmos padrões que os modelos de segurança tradicionais, enquanto se adapta aos detalhes dos ambientes nativos da nuvem, ou seja, mudanças rápidas de código e infraestrutura altamente efêmera. A segurança nativa da nuvem está altamente relacionada à prática chamada [DevSecOps](/devsecops/).
```

**Review by jessicalins — COMMENTED** [review]
looking good, left a few suggestions (link fixes + name consolidation)

**Comment by edsoncelio** [comment]
> looking good, left a few suggestions (link fixes + name consolidation)

Thanks!! I'll open a PR to fix that before go to live :D

**Review by brunoguidone — APPROVED** [review]
LGTM

**Review by seokho-son — APPROVED** [review]
LGTM :)

**Pull request merged**